scanpipe/tests/data/is-npm-1.0.0_scan_codebase.json

```diff
@@ -68,14 +68,38 @@
       "sha256": "",
       "sha512": "",
       "copyright": "",
-      "license_expression": "mit",
-      "declared_license": "['MIT']",
+      "holder": "",
+      "declared_license_expression": "mit",
+      "declared_license_expression_spdx": "MIT",
+      "license_detections": [
+        {
+          "matches": [
+            {
+              "score": 100.0,
+              "matcher": "1-spdx-id",
+              "end_line": 1,
+              "rule_url": null,
+              "start_line": 1,
+              "matched_text": "MIT",
+              "match_coverage": 100.0,
+              "matched_length": 1,
+              "rule_relevance": 100,
+              "rule_identifier": "spdx-license-identifier-mit-5da48780aba670b0860c46d899ed42a0f243ff06",
+              "license_expression": "mit"
+            }
+          ],
+          "identifier": "mit-a822f434-d61f-f2b1-c792-8b8cb9e7b9bf",
+          "license_expression": "mit"
+        }
+      ],
+      "other_license_expression": "",
+      "other_license_expression_spdx": "",
+      "other_license_detections": [],
+      "extracted_license_statement": "['MIT']",
```

Comment:
This is a bug!! we should not have this ugly repr-like in the extracted license statement but proper YAML like this was done before with the SCTK 31.x in "declared_license" . @AyanSinhaMahapatra ping :)